docs/visual-basic/programming-guide/concepts/caller-information.md

```diff
@@ -73,6 +74,19 @@ End Sub
 |Attribute constructor|The name of the member to which the attribute is applied. If the attribute is any element within a member (such as a parameter, a return value, or a generic type parameter), this result is the name of the member that's associated with that element.|  
 |No containing member (for example, assembly-level or attributes that are applied to types)|The default value of the optional parameter.|  
   
+### <a name="CALLEREXPRESSION"></a> Caller expressions  
+
+You use the <xref:System.Runtime.CompilerServices.CallerArgumentExpressionAttribute?displayProperty=nameWithType> when you want the expression used for an argument. Diagnostic libraries may want to provide more details about the *expressions* used for arguments. By providing the expression that triggered the diagnostic, in addition to the parameter name, developers see more details about the code that triggered the diagnostic. That extra information makes it easier to fix. The following method uses the <xref:System.Runtime.CompilerServices.CallerArgumentExpressionAttribute> to display the condition that must be `True`:
```

Comment:
Again, I'm not sure users will relate to "used for" as much as "passed..." I think folks think of passing arguments around before arriving at a method, in which they are used.